module/zfs/zfs_fm.c

```diff
@@ -778,15 +778,16 @@ zfs_ereport_post(const char *subclass, spa_t *spa, vdev_t *vd, zio_t *zio,
 	nvlist_t *ereport = NULL;
 	nvlist_t *detector = NULL;
 
+	/* If this is a rate limiting event, don't do any work */
```

Comment:
Normally I'm all for comments.  But I don't think this adds anything which isn't already in the function name.